src/pr/TestReportRenderer.js

```diff
@@ -62,6 +65,101 @@ class TestSuite extends Component {
   }
 }
 
+class TestSummary extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showDetail: false,
+    };
+  }
+
+  render() {
+    const toggle = () => {
+      this.state.showDetail = !this.state.showDetail;
+      this.setState(this.state);
+    };
+    const iconStyle = { cursor: "pointer" };
+    let icon = <BsFillCaretRightFill style={iconStyle} onClick={toggle} />;
+    if (this.state.showSuites) {
+      icon = <BsFillCaretDownFill style={iconStyle} onClick={toggle} />;
+    }
+    let suiteDetail = null;
+    if (this.state.showDetail) {
+      let rows = [];
+
+      let data = [];
+      for (const [filename, classes] of Object.entries(this.props.info)) {
+        for (const [classname, stats] of Object.entries(classes)) {
+          stats.filename = filename;
+          stats.classname = classname;
+          data.push(stats);
+        }
+      }
+      data = data.sort((a, b) => {
+        a = a.passed + a.error;
+        b = b.passed + b.error;
+        if (a === b) return 0;
+        return a > b ? -1 : 1;
```

Comment:
Aren't those equivalent?
```suggestion
        return b - a;
```

Reply:
> Aren't those equivalent?